packages/react-storage/src/components/StorageBrowser/Views/LocationDetailView/Controls.tsx

```diff
@@ -86,7 +87,7 @@ export const LocationDetailViewControls = (): React.JSX.Element => {
       <Paginate />
       <LocationDetailMessage />
       <Loading />
-      <LocationDetailViewTable />
+      <DataTableControl />
```

Comment:
Can we delete `LocationDetailViewTable` with this change?

Reply:
yes we can, was waiting for Heather's changes for pagination to get in first